projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/onboarding-guide/index.tsx

```diff
@@ -0,0 +1,32 @@
+// import { WpcomTourKit } from '@automattic/tour-kit';
+// import type { WpcomConfig } from '@automattic/tour-kit';
```

Comment:
This is commented out because I am getting build errors when trying to use this packages,  I have not figured out why yet